doc/benchmarking.md

```diff
@@ -43,6 +42,8 @@ during encoding.
     context clustering.
 *   `kitten` optimizes the adaptive quantization for a psychovisual metric.
 *   `tortoise` enables a more thorough adaptive quantization search.
+*   `glacier` disables chunked encoding and enables whole image optimizaions.
+*   `tectonic plate` exhaustively tries different glacier settings.
```

Comment:
"tectonic_plate", also description is inaccurate, just tries multiple glacier configurations. Maybe instead of duplicating all these descriptions, this doc should just link to encode_effort.md.